docs/pages/ops-guide/overview.md

```diff
@@ -352,6 +352,20 @@ Placement constraints can be applied to zones by referring to the `@zone` key. F
 
 When the region awareness feature is enabled (currently in beta), the `@region` key can also be referenced for defining placement constraints. Any placement constraints that do not reference the `@region` key are constrained to the local region.
 
+#### Rack aware services
+
+Many rack aware services do not support reliable migration between rack-aware and non-rack-aware operation.  These services can choose to restrict the set of placement constraint transitions which are valid through use of the `ZoneValidator`.  The allowed placement constraint transitions when using the `ZoneValidator` are as follows:
+```
+Placement Constraint References Zones
+
+| Original Constraint | New Constraint |
```

Comment:
I think there's native support for showing tables as HTML if you like, see e.g.:

rendered: https://github.com/mesosphere/dcos-commons/blob/master/frameworks/hdfs/docs/quick-start.md
raw: https://raw.githubusercontent.com/mesosphere/dcos-commons/master/frameworks/hdfs/docs/quick-start.md